### ISSUE TITLE
[BUG] Lunr search doesn't work in non-default language

### ISSUE THREAD

**Issue by nekr0z** [dillonzq/LoveIt#403]
With Lunr search enabled, search works as expected in default language.

If switched to another language on a multilingual site, search does not work. Attempts to enter search term in the search feed result in
```
TypeError: Cannot read property 'apply' of undefined
    at e.Builder.use (lunr.min.js:6)
    at e.Builder.<anonymous> (theme.js:219)
    at e (lunr.min.js:6)
    at theme.js:218
```
in the browser console.

**Comment by py563** [comment]
is this on your website? then above mentioned error message is displayed only in English version search not on default language which I guess is Russian on your site.

**Comment by nekr0z** [comment]
Yes, that's on my website (it runs a heavily customized LoveIt, but I get same results with current master of LoveIt).

Lunr is strange, it only searches for the last word in the input, but for me in Russian it does so reliably. It doesn't find any non-cyrillic words in Russian version of my site (although there are such words), though, but it gives no error in this case.

In the English version it always gives error.
